File: Backend/Mono/Nonagon.Modular.DynamicForm/FormLayoutType.cs
using System;

namespace Nonagon.Modular.DynamicForm
{
	/// <summary>
	/// Form layout type.
	/// </summary>
	public enum FormLayoutType
	{
		/// <summary>
		/// The linear layout. All children will be stacked in the specified Orientation.
		/// </summary>
		LinearLayout,

		/// <summary>
		/// The wrap layout. All children will be stacked just like linear layout. 
		/// But will also wrap to next row if overflow.
		/// </summary>
		WrapLayout
	}
}

File: Frontend/GWT/gwt-modular-dynamicform/src/com/nonagon/modular/dynamicform/client/FormLayoutType.java
package com.nonagon.modular.dynamicform.client;

public enum FormLayoutType {

	LinearLayout,
	WrapLayout;
	
	public static FormLayoutType fromString(String value) {

		if(value == null)
			return LinearLayout;
		
		return Enum.valueOf(FormLayoutType.class, value);
	}
	
	public static String fromEnum(FormLayoutType formLayoutType) {
		
		return formLayoutType.toString();
	}
}
